spec/models/vacancy_spec.rb

```diff
@@ -551,100 +490,6 @@
     end
   end
 
-  context "publishers can set certain attributes under certain conditions" do
-    context "when the vacancy is at the central office of a trust" do
-      subject { build_stubbed(:vacancy, :central_office) }
-
-      it_behaves_like "allows the user to set the education phase of the vacancy"
-    end
-
-    context "when the vacancy itself has a phase" do
-      subject { build_stubbed(:vacancy, phase: phase) }
-
-      context "when the phase is primary" do
-        let(:phase) { "primary" }
-
-        it_behaves_like "does not allow the user to set the subjects"
-
-        it_behaves_like "allows the user to set the key stage"
-      end
-
-      context "when the phase is secondary" do
-        let(:phase) { "secondary" }
-
-        it_behaves_like "allows the user to set the subjects"
-
-        it_behaves_like "does not allow the user to set the key stage"
-      end
-    end
```

Comment:
Are we not going to test for any of this now?  The two methods below still determine which attributes can be set, right? Don't we want to test them?

```
def allow_subjects?
    phases.any? { |phase| phase.in? %w[middle secondary through] }
end
```

```
def allow_key_stages?
    phases.any? { |phase| phase.in? %w[primary middle secondary through] }
end
```